def test_force_fail():
    assert False
